src/autoscaler/envelopeprocessor/envelopeprocessor_test.go

```diff
@@ -68,17 +68,19 @@ var _ = Describe("Envelopeprocessor", func() {
 
 		Context("processing container metrics", func() {
 			BeforeEach(func() {
-				envelopes = append(envelopes, generateContainerMetrics("test-app-id", "0", 10.2, 10*1024*1024, 20*1024*1024, 1111))
-				envelopes = append(envelopes, generateContainerMetrics("test-app-id", "1", 10.6, 10.2*1024*1024, 20*1024*1024, 1111))
+				envelopes = append(envelopes, generateContainerMetrics("test-app-id", "0", 10.2, 50, 10*1024*1024, 20*1024*1024, 1111))
+				envelopes = append(envelopes, generateContainerMetrics("test-app-id", "1", 10.6, 51, 10.2*1024*1024, 20*1024*1024, 1111))
 				envelopes = append(envelopes, generateMemoryContainerMetrics("test-app-id", "2", 10.2*1024*1024, 1111))
 				envelopes = append(envelopes, generateMemoryQuotaContainerMetrics("test-app-id", "2", 20*1024*1024, 1111))
+				envelopes = append(envelopes, generateCPUContainerMetrics("test-app-id", "3", 1, 1111))
```

Comment:
unrelated bugfix: here is the test for https://github.com/cloudfoundry/app-autoscaler-release/pull/2790/files#r1535346958 to avoid regression